runtime/src/iree/hal/drivers/vulkan/CMakeLists.txt

```diff
@@ -118,3 +118,12 @@ iree_cc_test(
 )
 
 ### BAZEL_TO_CMAKE_PRESERVES_ALL_CONTENT_BELOW_THIS_LINE ###
+
+# If renderdoc support is enabled we can make use of it in the device.
+# Note that we disable this by default as it introduces a backdoor.
+if(IREE_ENABLE_RENDERDOC_PROFILING)
+  target_compile_definitions(iree_hal_drivers_vulkan_vulkan
+    PUBLIC
+      "IREE_HAL_VULKAN_HAVE_RENDERDOC=1"
+  )
+endif()
```

Comment:
Thanks for making this CMake-only and optional. That will simplify pulling this into the internal Bazel repo.

(other context: https://github.com/iree-org/iree/pull/6207 removed our prior `third_party/renderdoc/` configuration)